MiniAn/minian_main.py

```diff
@@ -572,7 +682,7 @@ def save_minian_to_doric(
 
         print(mn_defs.Messages.SAVE_ROI_SIG, flush=True)
         pathROIs = vpath+mn_defs.DoricFile.Group.ROISIGNALS+operationCount+'/'
-        utils.save_roi_signals(C.values, A.values, time_, f, pathROIs+vdataset, attrs_add={"RangeMin": 0, "RangeMax": 0, "Unit": "AU"})
+        utils.save_roi_signals(C.values, A.values, time_, f, pathROIs+vdataset, attrs_add={"RangeMin": 0, "RangeMax": 0, "Unit": "AU"}, roi_ids=A.coords["unit_id"].values)
```

Comment:
We have to make sure that unit ids start from 1 not 0, before calling the function `save_minian_to_doric()`?

Reply:
@kapilseh do they start from 1? even if the cross-reg isn't used

Reply:
In the minian it starts from 0 (even without crossReg).
I wanted to change the unit_id in the function initialize_components.
But unit_id changes to start with 0 in cnmf_1
so made the changes in cnmf_1.
This seems fine. But I think - will have to do a similar change for the preview as well.
@4D42 @doriclenses

Reply:
It has to happen before -- in `initialize_seeds()`, we probably can renumber `seeds_final` there, or if not possible, in the `initialize_components()`